backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsync.java

```diff
@@ -121,7 +123,10 @@ public CompletableFuture<Set<Person>> savePatients(byte[] content, UUID facility
         // only increment count after person record is created
         patientCount++;
 
-        if (!patientsList.contains(newPatient)) {
+        // UGH iI need to change this logic too, because patients could be batched in different
```

Comment:
did you mean to leave this comment in?

Reply:
I certainly did not 😆 Thanks for catching that! Removed